theme/styles/snippets/product-description.scss

```diff
@@ -0,0 +1,84 @@
+.ProductDescription {
+  padding-bottom: 3.125rem;
+
+  h1 {
```

Comment:
We generally prefer classes over referencing elements by name. This way we can easily change the element without having to worry about breaking styles :)